src/pages/ThorChainLP/YourPositions.tsx

```diff
@@ -93,6 +94,24 @@ export const YourPositions = () => {
           const { assetId } = useMemo(() => position, [position])
           const poolAssetIds = useMemo(() => [assetId, thorchainAssetId], [assetId])
 
+          const positionStatusTag = useMemo(() => {
+            if (position?.status.incomplete) {
+              return (
+                <Tag colorScheme='red'>
+                  <Text translation='common.incomplete' />
+                </Tag>
+              )
+            }
+
+            if (position?.status.isPending) {
+              return (
+                <Tag colorScheme='yellow'>
+                  <Text translation='common.pending' />
+                </Tag>
+              )
+            }
```

Comment:
Can we return null here as a failsafe ? Or as an early return if `(!position)`

Reply:
https://github.com/shapeshift/web/pull/6481/commits/8ee8e29363ff61d25d76cdca43c91085fdad40a6